src/Symfony/Component/Messenger/Worker.php

```diff
@@ -46,11 +49,12 @@ class Worker
     private WorkerMetadata $metadata;
     private array $acks = [];
     private \SplObjectStorage $unacks;
+    private ?ContainerInterface $rateLimiterLocator;
 
     /**
      * @param ReceiverInterface[] $receivers Where the key is the transport name
      */
-    public function __construct(array $receivers, MessageBusInterface $bus, EventDispatcherInterface $eventDispatcher = null, LoggerInterface $logger = null)
+    public function __construct(array $receivers, MessageBusInterface $bus, EventDispatcherInterface $eventDispatcher = null, LoggerInterface $logger = null, ContainerInterface $rateLimiterLocator = null)
```

Comment:
Can we pass an array as we do for `$receivers`? That would be more consistent and easier to use when not using the full stack framework.

Reply:
Yes, I do not see why not!